lmdeploy/cli/serve.py

```diff
@@ -139,6 +139,16 @@ def add_parser_api_server():
                             type=str,
                             default=None,
                             help='The proxy url for api server.')
+        parser.add_argument(
+            '--concurrency-pressure',
```

Comment:
--max-concurrency, default None, type int. 
Let users decide whether to enable it or an appropriate value.

Reply:
cc @lzhangzz
any comments?

Reply:
> --max-concurrency, default None, type int. Let users decide whether to enable it or an appropriate value.

How about `--max-concurrent-requests` in TGI?

Reply:
Sure.